src/notifications/ManagerMac.mm

```diff
@@ -106,10 +106,18 @@ + (UNNotificationAttachment*)createFromImageData:(NSData*)imgData
                                                                                 textInputButtonTitle:sendStr.toNSString()
                                                                                 textInputPlaceholder:placeholder.toNSString()];
 
-    UNNotificationCategory* category = [UNNotificationCategory categoryWithIdentifier:@"ReplyCategory"
-                                                                              actions:@[ replyAction ]
-                                                                    intentIdentifiers:@[]
-                                                                              options:UNNotificationCategoryOptionNone];
+    UNNotificationCategory* category;
+    if(!room_id.isEmpty()){
+        category = [UNNotificationCategory categoryWithIdentifier:@"ReplyCategory"
+                                                          actions:@[ replyAction ]
+                                                intentIdentifiers:@[]
+                                                          options:UNNotificationCategoryOptionNone];
+    }else{
+        category = [UNNotificationCategory categoryWithIdentifier:@"ReplyCategory"
+                                                          actions:@[]
+                                                intentIdentifiers:@[]
+                                                          options:UNNotificationCategoryOptionNone];
+    }
```

Comment:
I don't know Objective-C(++), but is there any chance that this could be simplified at all (e.g. temp variable for `actions` or just a ternary statement)?

Reply:
I agree, that would be ideal, but I don't know Objective-C++ either. In fact, I struggle to understand what is the type of ``@[]``, and knowing this is necessary to create a temporary variable. Is it a NSArray? Or a [UNNotificationAction] (does this type actually exists or it is a documentation convention? The ``[]`` around it I mean)?